pyfmt/__init__.py

```diff
@@ -25,8 +27,121 @@
 ]
 
 
-def pyfmt(path, check=False, line_length=100, extra_isort_args="", extra_black_args="") -> int:
+def pyfmt_title():
+    """
+    Display a cool ascii art title
+    """
+    pyfmt_splash = [
+        "               __           _    ",
+        "              / _|         | |   ",
+        "  _ __  _   _| |_ _ __ ___ | |_  ",
+        " | '_ \| | | |  _| '_ ` _ \| __| ",
+        " | |_) | |_| | | | | | | | | |_  ",
+        " | .__/ \__, |_| |_| |_| |_|\__| ",
+        " | |     __/ |                   ",
+        " |_|    |___/                    ",
+    ]
+    print("\033[94m")
+    for part in pyfmt_splash:
+        print(f"{part.center(os.get_terminal_size().columns, ' ')}")
+    print("\033[0m")
+
+
+def find_all_files_and_dirs():
+    """
+    Map out all files and directories in the current
+    working directory
+    """
+    all_files = []
+    all_dirs = []
+    for root, dirs, files in os.walk("."):
+        for name in files:
+            # Saving filename
+            all_files.append(name)
+        for name in dirs:
+            # Saving directory name
+            all_dirs.append(os.path.abspath(os.path.join(root, name)))
+    # Remove duplicates
+    all_files = list(set(all_files))
+
+    return all_files, all_dirs
+
+
+def display_divider(title="", character="=", color_code="\033[94m"):
+    """
+    Divider between sections of program.
+    """
+    if title:
+        print(
+            "\n"
+            + "{}".format(color_code)
+            + "  {}  ".format(title.upper()).center(
+                os.get_terminal_size().columns, "{}".format(character)
+            )
+            + "\033[0m"
+        )
+    else:
+        print(
+            "\n"
+            + "{}".format(color_code)
+            + "".center(os.get_terminal_size().columns, "{}".format(character))
+            + "\033[0m"
+        )
+
+
+def pyfmt(
+    path, skip="", check=False, line_length=100, extra_isort_args="", extra_black_args=""
+) -> int:
     """Run isort and black with the given params and print the results."""
+    pyfmt_title()  # Display title
```

Comment:
while I like the ascii art you made, but I'm not suree I want to see a title everytime pyfmt runs. it will fill up my terminal with spam since I run it so often

Reply:
i don't know seemed neat when i was doing it haha ... again, subjective, i'm good with leaving it out or replacing it with something more compact

Reply:
at the very least if you could make it an opt-in sort of thing, like if the user passes `--show-title` or something it shows but otherwise it doesn't

Reply:
yeah i can do that

Reply:
yeah i can do that